src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,162 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private static class Node<T> {
+        private T element;
+        private Node<T> first;
+        private Node<T> last;
+
+        public Node(Node<T> first, T element, Node<T> last) {
+            this.element = element;
+            this.first = first;
+            this.last = last;
+        }
+    }
+
+    private Node<T> head;
+    private Node<T> tail;
+    private int sizeList = 0;
```

Comment:
in the class order elements this way : fields -> init blocks -> constructors -> methods / inner class